### PR TITLE
14159 Disabled conversion corrections + cleanup

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#14159

*Description of changes:*
- disabled Conversion corrections
- un-disabled Transition corrections (now falls into "others enabled by default")
- updated unit tests
- app version = 5.7.16

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/FilingHistoryList.vue [hunk 1]
```diff
@@ -703,17 +703,20 @@ export default class FilingHistoryList extends Vue {
...
       case FilingTypes.CORRECTION:
       case FilingTypes.REGISTRATION:
         // correction via Edit UI
         this.currentFiling = item
         this.fileCorrectionDialog = true
         break
 
+      case FilingTypes.ANNUAL_REPORT:
+      case FilingTypes.CHANGE_OF_ADDRESS:
+      case FilingTypes.CHANGE_OF_DIRECTORS:
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList.vue:719]: These are redundant but I added them to show that we explicitly can correct them.

File: src/components/Dashboard/FilingHistoryList.vue [hunk 2]
```diff
@@ -872,18 +875,18 @@ export default class FilingHistoryList extends Vue {
...
 
-    // any cases not listed below are allowed (enabled)
+    // list of cases to disable correction
+    // (any case not listed below is ALLOWED)
     conditions[0] = () => !this.isAllowed(AllowableActions.FILE_CORRECTION)
     conditions[1] = () => item.availableOnPaperOnly
     conditions[2] = () => item.isTypeStaff
     conditions[3] = () => item.isFutureEffective
     conditions[4] = () => this.isTypeAlteration(item)
-    conditions[5] = () => this.isTypeTransition(item)
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList.vue:881]: Allowed by default.

File: src/components/Dashboard/FilingHistoryList.vue [hunk 3]
```diff
@@ -872,18 +875,18 @@ export default class FilingHistoryList extends Vue {
...
+    conditions[5] = () => this.isTypeConversion(item)
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList.vue:885]: Explicitly not allowed (disabled).

**Review by lewischenstudio — APPROVED** [review]
LGTM

**Review by ChrisGabel23 — APPROVED** [review]

**Review by ketaki-deodhar — APPROVED** [review]

File: src/components/Dashboard/FilingHistoryList.vue (hunk 3 above)
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList.vue]: Obsolete comment -- conversion local corrections are allowed.

**Pull request merged**